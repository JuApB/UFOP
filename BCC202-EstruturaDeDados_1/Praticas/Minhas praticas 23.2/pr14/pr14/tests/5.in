5
2638 29 33 52 19 47 40 73 61 56 24 93 82 69 26 81 76 74 66 84 7 74 78 64 36 100 16 1 53 57 72 
353 vetar estulticia intra dildo estar jazia viram romper fortuito dona inteirar arigo surto honorarios deus messe erudito rever flagelo sinergia carater crise temer avante invocar estatico familia respaldar seda pirou purgar zuado prazo afins atraso pusilanime anedota cera cingir permanente densa postergado iogue alude natal executado evadir devido hiper abono abundante bagar dispensar fato assolar saude hobby unilateral safra vale defasagem trazer perai enobrecer efebo amiga todo vulgo provimento frase ente obstinado galga excitar crime aflito utensilio ousado apaziguar convir penal pestanejar remessa fores otica condiz tempestivo finda danca as vertente viver ma-fe nescio balir relevante difundido tenra conhecer doutrina indelevel aceso moda puxou cuidar coincidir devoto braba slide rito mas jurista fascinar trivial roto sumir reino modestia adquirir promulgar martir lavor ator amago odor escassez emergente junto duro pleito leria dolar possuir empafia obsta minar sope nigga tiete trevas gestao dominio opiniao furto penha gentil servo tratar enigma agravante de veridico restringir desavenca pseudo tumba beleza assimilar labaro proporcao fusao fantastico pedante ativismo valor trino nativo discutir flor grata bula excitado celo mores sentido venal curso cobro viger instancia perscrutar preencher houve turbe prelo efemero manjo inchado segregacao proceder ciencia especifico perpetrar vou dread diferir pudor exodo empenho derradeiro prosa interesse zambi astronomia desvanecer austero imerso uncao volupia candeia gorar revogado opressor consagrar leao poeta costume desde dialogo intenso abdicar bola polir eficaz juncao pifio melindroso polidez forca efeito maxima ramal honesto conta dele categoria sectario ciume cite extirpacao suite relar caril lexico decisao delicia redundante genocidio senso poder pane emanar idoneidade soldo extenso reverberar tezao demagogo na difusao lamentavel estara turvo ecran possivel sahel causa para aspero elogio dolo pemba aprisco recorrente consigamos precedente azia mesma viabilizar deslumbrar residencia historia hibrido expurgar darao loja sintese penhor denominado pensar alento conceder feita esbornia pegos perverso sancao pouco veaco vesga rapaz foque desafio faccioso enfoque paradigma inercia saia descer carcere degradante crica aprimorar desempenho mencao orbe conflito saliente extorquir baila comeco reprobo suprimir blasfemia ignorancia cabal point mesmo proposito errante grafar passo local cesso ruco exige imaginacao porventura kombi compensar bolei exemplo valer soberano alto isso discernir queca bodar exiguo discricao tangivel coeso 
4409 77 14 31 78 4 15 4 23 57 44 36 36 49 78 51 67 34 6 11 44 53 97 39 22 68 55 71 66 89 79 41 7 73 78 95 30 9 42 25 50 38 39 87 89 100 59 19 12 2 4 61 3 44 89 70 80 20 15 74 79 97 32 60 43 75 57 1 88 5 65 6 70 48 91 92 12 3 60 
850 pseudo anil verdadeiro argumento gabar amor eminencia pensar alveo dirimir romper sope dissabor pedi regular recorrente reprobo tolerancia rola eficaz todavia exodo ramal residencia volupia pazes fome acha protelar triste coracao difusao escassez exemplo lexico coice densa otica sentido lume afeto queda deus ponto parecer dialogo mentor aceso alude vou asceta pedante inteirar remontar crica contumaz moda oportuno exequivel minar desercao pressupor degradante estulticia passo beleza perfeito proscrito ativismo perai restringir felicidade insular abada diferir adquirir roto conviccao palio dispendio aguerrido ardil mas mensuracao agravante chape genocidio respaldar fragil sahel descer versa cetico zuado seria fantastico titubear outrora alento cacou opoem demagogo real ruco labaro remoto perua exprimir dende assolar morfologia finda conotativo discorrer causa pifio latifundio ainda viram aprisco preciso gratidao mora opulencia proporcao fuark rico ecran inter relar conceito oriundo repugnante abordar efemero evoe mesma compor postergado deboche empenho rescindir obice habito conflito foyer qualquer ente prega goma grata zonza poeta xingar bagar fatalidade grife leito houve imerso leviandade perda macula passar juncao enfoque biota grotesco tezao litigio nigga vago emanar blefe afins veado vicar efebo forca procedente dread viger balela tempestivo jegue ancestral imparcial arigo desvanecer babau outrossim dolo ponha assertivo repulsa cuidar prazo estara comensal interacao capo surto obstinado acronimo casal pirou suave pode intenso desavenca esplendor dolar foco macambuzio baque coibir aspero erudito junto categoria patim parir excitado nessa saia vigente compensar manter esquisito frisar anedota analise conceder especifico melindroso item cedeu agouro desempenho ambas despota porventura vangloriar vital fabuloso sinergia defasagem condiz mores igapo enquanto acao hiper mister capacidade facho abate aflito redundante findar povo preencher trazer curso sairam cobica diccao impar satisfeito tangivel cheio dotar fato admoestar paria letal podem timido sucedido parabola sancao rito celo jiboia alarido supressao lamentavel saliente golfo promulgar maior lousa resolucao ator prova moribundo importar amago provimento outrem busto sebe alta cera penha inercia intuitivo super vetar brown devaneio sedentario dominio derradeiro denominado munido safra balizar polidez simpatico difundido boca estimativa aspirar credulo sectario reino perpetrar resoluto concatenar viver bravo estorvo interesse bar historia soldo mexer ademais basal fuder curro pelo optar encher apos atroz uncao amplo feita faceta cedo abono campo fortuito arco prima leviano honorarios imponente dose alelo lugar imaginacao sobrinha suite apaziguar ilusao banir ma-fe tiziu mouro fucar preterido dotes mini sacas explanar discutir observar ajayo poder reverberar indez cessao passivel arfar singular queca diligente infringir foque atraso desdem arauto deflagrar tipico pesa expiar fascinar sesta caibo perpassar jovial calmo varei periodico vesga nanar comeco amiga voto intrinseco confidente nicho respectivo tese mongo escoria zelo messe delicia zambi revogado vira igneo vestigio associacao gato paradigma aflicao presumido extirpacao vulgo detentor conta maroto danca carater relutante sumir soturno bagui aduzir errante dignificar galga deslumbrar sabao vedar horda galhardia dispersar loja avidez fulgor poente honesto todo tumba jazer opiniao pemba mencao fusao manar componente avaro patente tenra remessa local relapso limpo as ativa purgar servo trivial vulto retorica humo mambo ponte retaguarda iogue ponca etimologia esbornia vieram submergir lida atentar point civil dentre onus obsta coeso sobre crise incluir enobrecer seda baila blasfemia saudoso avante extrinseco indelevel inchado definicao fidalgo rapaz viabilizar devoto condao trevas abdicar isso slide alto martir estoicismo imbecil miose autoestima portanto para axial avoar bodar cabra caril ficha modestia sega mamas guia expedido conjuracao afeccao index devido desuso execucao hibrido gueto autarquia orbe chabu espectro invocar pouco vivaz sigla natal crime acesa vale atem dispensar balir acai pessoa leria ignorancia pestanejar descritivo pudor taba tipo culminar estrito regozijo doutrina empafia ocio unilateral coincidir patetico duro convir alteridade ermitao contracapa siso ciencia cobro elegivel probatorio desafio avarento maneira ripar tenso hermetico mundo prestativo gambe rever categorico executado vigencia cutis proceder costume tratar fungo lavor veridico ocasiao acre mesmo flor estatico asseado dubio correcao exige requerer ostracismo odor irmao maxima trem velar viado flagelo contiguo coercao designado fluir constatar logre pleito sisudo elogio estar dialetica pesquisa alado declinio evadir isento veaco dildo na assimilar xiita exiguo manjo ciume digna derby nativo genro torar enigma cona ruim ranso reportar segregacao perverso rapido darao suprimir cuia assa coxo gentil matriz trino vertente amanha carcere gestao etnia venal perscrutar giria opressor hobby furto fado resma chara translado hipotetico prosa ignominia magro polir atender proposito desde ousado claro puxou braba frugal gnose transviado safam grafar mito senso focar austero solicitar ouvir possivel bola consagrar inibe bula politizado atitude lorde ajam cite apetecer fisionomia extravasar caos alienado desgraca conhecer humildade temer idoso instancia sabio ralou deusa pinar trafico imago tenda solene consigamos kombi bilro agem azia exato pegos dele pusilanime exilio formoso precedente pessoal dona pejorativo pane prelo reaver permanente pacato estio idoneidade ridiculo turvo candura maximo composicao extorquir familia astronomia cozer discernir jurista retem excitar arqui intra fomento decisao justificar de tiete opcao oxala amante faccioso frivolo receoso cabal valer penhor banzo deram liberdade valor dorso expurgar cingir penal virtu simbiose prescindir ruir abundante soube maloqueiro arrebatar esplendido efeito eminente aprimorar harem integracao moralismo frase lastima tento mitra arcaico cesso turbe bolei parreira outro vivo emergente ownar leao broxa extenso candeia relevante restaurar mentecapto fores utensilio possuir sintese cautela informacao discricao gorar futum saude jazia cacar nescio soberano sanidade 
1553 20 12 98 93 99 43 92 92 7 48 21 80 92 39 11 67 86 17 89 97 22 
16 paradigma empafia valor vou fato caril pedante delicia proceder kombi remessa danca penha volupia excitar pegos 
1636 5 23 67 80 17 64 
209 honorarios ente prazo fato nigga bagar manjo crise ecran atraso efebo difusao relevante galga segregacao juncao arigo dolo ma-fe caril denominado exodo bolei extirpacao trino desempenho pleito extorquir darao loja cera vou obstinado queca veridico porventura esbornia local agravante frase vetar volupia martir trevas mesmo forca otica fores todo excitado zambi proceder cite pouco doutrina cuidar penal idoneidade reprobo elogio fantastico demagogo baila purgar flor ruco exige poeta foque fusao valor possuir postergado deslumbrar degradante desvanecer cesso discutir empenho honesto convir melindroso sinergia venal pirou rever suprimir mas feita modestia maxima blasfemia slide respaldar leao alto zuado inteirar conceder houve devoto delicia tezao jurista anedota aprimorar curso odor sope para hiper defasagem ignorancia restringir densa diferir proposito aceso conta especifico ciume pensar amago balir pemba uncao comeco estulticia tiete moda flagelo emanar perpetrar pudor opiniao condiz faccioso sancao cingir viver pusilanime erudito perscrutar deus possivel familia vale sumir seda ousado utensilio kombi lavor errante senso empafia nativo revogado costume viram obsta estar nescio residencia sentido remessa executado assimilar bola exemplo fortuito messe imerso ciencia safra descer tangivel discricao penha perai pedante redundante enigma jazia bula hibrido paradigma exiguo decisao pegos estatico excitar viger saude danca grafar carcere alude coincidir orbe ator pseudo consagrar lexico aspero dolar reino efeito puxou 
19 9 99 72 78 76 6 57 15 35 89 
17 penha pedante fato cesso pegos delicia danca excitar empafia kombi vou volupia valor remessa caril paradigma proceder 
