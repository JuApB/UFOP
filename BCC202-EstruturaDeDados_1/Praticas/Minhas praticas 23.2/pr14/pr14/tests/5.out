2337
1518
1036
1578
8
